database-grants/terraform/dev/setup_keys.sh

```diff
@@ -3,6 +3,9 @@
 # Retrieve keys from your netrc, specified as
 # machine sfo2.digitaloceanspaces.com login KEY password SECRET
 # Source this file before running terraform commands
-netrc_string=$(grep digitalocean.dev.treetracker.org ~/.netrc)
+
+grep_string=$(cat backend.tf | grep endpoint | awk -F'=' '{print $2}' | awk -F'/' '{print $NF}' | sed 's/"//g')
+
+netrc_string=$(grep ${grep_string} ~/.netrc)
```

Comment:
@sanketnadkarni is this relevant to this issue (fix version), if not, please split it into another pull request, thank you!

A good practice of raising pull request: make it small and focused.

Reply:
this may not be directly related though affects the terraform working. 
i can add this into a subsequent PR though.